engine/src/main/java/org/terasology/rendering/opengl/FrameBuffersManager.java

```diff
@@ -67,52 +93,45 @@
     // this logic one32thScale would have to be named one1024thResolution and the otherwise
     // straightforward connection between variable names and dimensions would have been lost. -- manu3d
     private Dimensions fullScale;
-    private Dimensions halfScale;
-    private Dimensions quarterScale;
-    private Dimensions one8thScale;
-    private Dimensions one16thScale;
-    private Dimensions one32thScale;
 
     // Note: this assumes that the settings in the configs might change at runtime,
     // but the config objects will not. At some point this might change, i.e. implementing presets.
     private Config config = CoreRegistry.get(Config.class);
     private RenderingConfig renderingConfig = config.getRendering();
 
     private Map<String, FBO> fboLookup = Maps.newHashMap();
+    private Map<String, Integer> fboUsageCountMap = Maps.newHashMap();
 
     private PostProcessor postProcessor;
     private Set<FBOManagerSubscriber> fboManagerSubscribers;
+    private Map<String, FBOConfig> dynamicFBOConfigs;
+    private Map<String, FBOConfig> staticFBOConfigs;
+    private boolean isDynamicFBOsGenerated;
 
     public FrameBuffersManager() {
         // nothing to do here, everything happens at initialization time,
         // after GraphicState and PostProcessors have been set.
+        isDynamicFBOsGenerated = false;
```

Comment:
1. FBOs is plural so it should be `areDynamicFBOsGenerated` or perhaps even closer to plain English `dynamicFBOsHaveBeenGenerated`
2. booleans are false by default, so there is no need for this line I think.